tests/Feature/Lexers/SeederLexerTest.php

```diff
@@ -57,7 +46,7 @@ public function it_returns_seeders()
         $this->assertIsArray($actual['seeders']);
         $this->assertCount(1, $actual['seeders']);
 
-        $this->assertSame(['Post'], $actual['seeders']);
+        $this->assertEquals(['Post'], $actual['seeders']);
```

Comment:
Why was there a need for the less strict assertion?

Reply:
Believe this was left over from a different change I made and then removed.